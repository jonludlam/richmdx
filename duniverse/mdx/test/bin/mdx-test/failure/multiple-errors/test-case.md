MDX should be able to report several simple errors such as
misused label or syntax errors inside blocks.

<!-- $MDX pif=paf -->
```ocaml
# 1 + 1;;
```

<!-- $MDX foo=bar -->
```ocaml
let x = 1
```
