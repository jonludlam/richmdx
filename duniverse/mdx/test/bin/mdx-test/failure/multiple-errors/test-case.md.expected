[mdx] Fatal error: File "test-case.md", lines 4-7: invalid code block: `pif` is not a valid label.
[mdx] Fatal error: File "test-case.md", lines 9-12: invalid code block: `foo` is not a valid label.
